docs/startup/preview/index.md

```diff
@@ -527,4 +527,5 @@ Take note of the following issues with the embedded deployment feature for the 0
     * Number of Read/Writes
     * Leader reelections
     * more..
-* The LocalApplicationRunner does not yet support the low level API. This means you cannot use StreamTask with LocalApplicationRunner.
\ No newline at end of file
+* The LocalApplicationRunner does not yet support the low level API. This means you cannot use StreamTask with LocalApplicationRunner.
+* Currently, you can only run a single applicaiton with a single zookeeper. To avoid this limitation, you need to make sure your app.id is unique.
```

Comment:
We can just say that the "app.id" must be unique for all applications using the zk cluster.
If not, typoe: "application" in the first line, and maybe say "single application at a time"